scripts/install_third_party.py

```diff
@@ -24,6 +24,7 @@
 import zipfile
 
 import common
+from contextlib import closing
```

Comment:
contextlib is a standard python module, so I would do "import contextlib" above "import itertools", above. Then in the code, refer to contextlib.closing(...).